src/ucs/datastruct/ptr_array.c

```diff
@@ -206,3 +244,69 @@ void *ucs_ptr_array_replace(ucs_ptr_array_t *ptr_array, unsigned index, void *ne
     ptr_array->start[index] = (uintptr_t)new_val;
     return old_elem;
 }
+
+
+/*
+   Locked interface functions implementation
+*/
+
+void ucs_ptr_array_locked_init(ucs_ptr_array_locked_t *locked_ptr_array, uint32_t init_placeholder,
+                        const char *name)
+{
+    /* Initialize spinlock */
+    ucs_recursive_spinlock_init(&locked_ptr_array->lock, 0);
```

Comment:
Pls handle the return value from `ucs_recursive_spinlock_init()`
what about adding the return value in `ucs_ptr_array_locked_init()`

Reply:
Fixed.